contracts/libs/0.8.x/MerkleLib.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "@openzeppelin-4.7/contracts/utils/cryptography/MerkleProof.sol";
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title Art Blocks Merkle Library
+ * @notice [TO FILL OUT]
```

Comment:
When we fill this out, lets make sure we also have natspec comments for the `ProjectConfig` struct and all our methods here as well (ChatGPT is great at writing initial comments btw hah 😄).